trigger.config.ts

```diff
@@ -0,0 +1,22 @@
+import { defineConfig } from '@trigger.dev/sdk/v3';
+
+export default defineConfig({
+  project: 'proj_laigugtxozpzhsnwklfr',
```

Comment:
Should this be configurable via environment variable? Or will each instance use the same Trigger.dev project?

Reply:
I would say yes, it should be configurable. Even if all of our self-managed instances use the same project, people may be self-hosting.

Does the Trigger.dev side need any configuration that we should document somewhere, or will it just work with a bare Trigger.dev project using this config?